packages/lib-classifier/src/plugins/tasks/dropdown-simple/components/SimpleDropdownTask.spec.js

```diff
@@ -1,108 +1,83 @@
     })
-    
-    it('should have no initial value', function () {
-      const grommetSelect = wrapper.find('Select')
-      expect(grommetSelect.props()['value']).to.equal(undefined)
+
+    it('should render without any option selected', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      expect(inputEl.value).to.equal('')
     })
-    
-    it('should render the correct number of options', function () {
-      const grommetSelect = wrapper.find('Select')
-      const renderedOptions = grommetSelect.props()['options'] || []
-      expect(renderedOptions).to.have.length(6)
+
+    it('should render the full list of options', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      await user.click(inputEl)
+
+      const options = screen.getAllByRole('option')
+
+      // Grommet Select only renders up to 20 options in the test environment, SHOULD be 40
+      expect(options.length).to.equal(20 /*SimpleDropdownTaskMockMedium.options.length*/)
+      expect(within(options[0]).getByText(SimpleDropdownTaskMockMedium.options[0])).to.exist()
     })
   })
 
-  describe('with an annotation', function () {
-    let wrapper
-
-    before(function () {
-      annotation.update({
-        selection: 2,  // Corresponds to "Jaunde"
-        option: true,
-      })
-      wrapper = shallow(
-        <SimpleDropdownTask
-          annotation={annotation}
-          task={task}
-        />
-      )
+  describe('Long list of options', function () {
+    beforeEach(function () {
+      const LongOptionsStory = composeStory(LongOptions, Meta)
+      render(<LongOptionsStory />)
     })
 
-    it('should pass the selected annotation to the Select sub-element', function () {
-      const grommetSelect = wrapper.find('Select').first()
-      expect(grommetSelect.prop('value')['text']).to.equal('Jaunde')
+    it('should render without any option selected', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      expect(inputEl.value).to.equal('')
     })
-  })
 
-  describe('when disabled', function () {
-    let menu
-
-    before(function () {
-      annotation.update({
-        selection: 2,  // Corresponds to "Yellow"
-        option: true,
-      })
-      menu = shallow(
-        <SimpleDropdownTask
-          disabled
-          annotation={annotation}
-          task={task}
-        />
-      )
-      .find(Select)
-    })
+    it('should render the full list of options', async function () {
+      const inputEl = screen.getByPlaceholderText('SimpleDropdownTask.selectPlaceholder')
+      await user.click(inputEl)
+
+      const options = screen.getAllByRole('option')
 
-    it('should disable the select menu', function () {
-      expect(menu.prop('disabled')).to.be.true()
+      // Grommet Select only renders up to 20 options in the test environment, SHOULD be 200
```

Comment:
Same comment as above. Because Grommet does not let you truly test the LongOptionsStory, I recommend deleting its unit tests. They're just a replica of ShortOptionsStory unit tests.